test/test_mlmodel.py

```diff
@@ -0,0 +1,86 @@
+from carla.data.catalog import DataCatalog
+from carla.models.catalog import MLModelCatalog
+
+
+def test_properties():
+    data_name = "adult"
+    model_tf_adult = MLModelCatalog(data_name, "ann")
+    # TODO: Issue #16
+    # model_pt_adult = MLModelCatalog(data_name, "ann", ext="pt")
+
+    exp_backend_tf = "tensorflow"
+    # TODO: Issue #16
+    # exp_backend_pt = "pytorch"
+    exp_feature_order_adult = [
+        "age",
+        "fnlwgt",
+        "education-num",
+        "capital-gain",
+        "capital-loss",
+        "hours-per-week",
+        "workclass_Private",
+        "marital-status_Non-Married",
+        "occupation_Other",
+        "relationship_Non-Husband",
+        "race_White",
+        "sex_Male",
+        "native-country_US",
+    ]
+
+    assert model_tf_adult.backend == exp_backend_tf
+    # TODO: Issue #16
+    # assert model_pt_adult.backend == exp_backend_pt
+    assert model_tf_adult.feature_input_order == exp_feature_order_adult
+    # TODO: Issue #16
+    # assert model_pt_adult.feature_input_order == exp_feature_order_adult
```

Comment:
If we have an issue, should we rather put it in the issue to add the respective tests, instead of putting all these comments in the source code?

Reply:
Yeah that should make more sense 😄